ui/react-ui/src/components/app/App.js

```diff
@@ -18,6 +19,7 @@ function App() {
         {/* Farhad */}
         <Route exact path='/users/:userId/tasks' component={UserTasks}/>
         <Route exact path='/reports/userId' component={userIdReport}/>
+        <Route exact path='/report/users/{user_id}/tasks' component={TasksReport}/>
```

Comment:
So for some reason, when i was making my endpoint, I went this deep with '/' and i was not able to fetch the data. I had to shorten the link from /reports/users/userid/tasks/taskid to reports/taskId. I am still not sure what the error was but it worked. Secondly, I am not able to fetch the report. Maybe its the CORS issue but I did not run into nor did @zxkevinxz .

Reply:
I see now your table is displaying so I think the issue is resolved. TBH I am also not sure on the short link thing. Furthermore I see you commented out the CORS line and it isnt having any issues. So I think this is good to go now